mifosng-android/src/main/java/com/mifos/mifosxdroid/passcode/PassCodeActivity.java

```diff
@@ -19,6 +20,7 @@ public int getLogo() {
 
     @Override
     public void startNextActivity() {
+        Toast.makeText(this, "Logged in Successfully", Toast.LENGTH_SHORT).show();
```

Comment:
Please put the hardcoded string in the strings.xml file and call from there.

Reply:
@RajaVamsi11 done

Reply:
@theabhishekavi Nice.